.cloudtest.yaml

```diff
@@ -13,183 +13,197 @@ import:
 reporting:
   junit-report: "results/junit.xml"
 executions:
-  - name: "Single cluster tests"
+  - name: "Single cluster tests uses shared nsm infrastructure"
     env:
-      - STORE_POD_LOGS_IN_FILES=true
-      - STORE_POD_LOGS_DIR=/home/circleci/project/.tests/cloud_test/$(cluster-name)
+      - PROCESS_ARTIFACTS_IN_TO_ARCHIVE=true
```

Comment:
PROCESS_ARTIFACTS_INTO_ARCHIVE? 
ARCHIVE_ARTIFACTS?

Reply:
sure.
PR has mark work in progress